src/transforms/normalize.js

```diff
@@ -1,31 +1,288 @@
+import warning from '../utils/warning'
+import { default as defaultSchema } from '../plugins/schema'
+import Normalize from '../utils/normalize'
+import { Map } from 'immutable'
+
+// Maximum recursive calls for normalization
+const MAX_CALLS = 50
+
+/**
+ * Refresh a reference to a node that have been modified in a transform.
+ * @param  {Transform} transform
+ * @param  {Node} node
+ * @return {Node} newNode
+ */
+
+function _refreshNode(transform, node) {
```

Comment:
Can you change these private functions to not be prefixed with `_`, and instead just have them at the end of the file? (For consistency with other places we do this)